.editorconfig

```diff
@@ -598,11 +598,6 @@ dotnet_diagnostic.SA1633.severity = none # FileMustHaveHeader
 dotnet_diagnostic.SA1642.severity = none # ConstructorSummaryDocumentationShouldBeginWithStandardText
 dotnet_diagnostic.SA1649.severity = none # FileNameMustMatchTypeName
 
-# Requires C# 8/9 - TODO Consider enabling
-dotnet_diagnostic.SA1141.severity = none # UseTupleSyntax
-dotnet_diagnostic.SA1316.severity = none # TupleElementNamesShouldUseCorrectCasing
-dotnet_diagnostic.SA1414.severity = none # TupleTypesInSignaturesShouldHaveElementNames
```

Comment:
I thought the idea was to list all available options, even if we set the severity to the default?

Reply:
That's the setup for the IDExxxx rules. For the stylecop ones we currently have those enabled en-masse and the turn off individual rules.